daemon/logger/awslogs/cwlogsiface_mock_test.go

```diff
@@ -49,6 +60,13 @@ func (m *mockcwlogsclient) PutLogEvents(input *cloudwatchlogs.PutLogEventsInput)
 	return output.successResult, output.errorResult
 }
 
+func (m *mockcwlogsclient) CreateLogGroup(input *cloudwatchlogs.CreateLogGroupInput) (*cloudwatchlogs.CreateLogGroupOutput, error) {
+	// m.createLogGroupArgument <- input
```

Comment:
Rather than commenting out the implementation of the mock, it'd be better to adjust the expectations set in the existing unit tests to handle the new calls to `CreateLogGroup`.